packages/web/components/transactions/transaction-details-slideover.tsx

```diff
@@ -24,10 +26,9 @@ export const TransactionDetailsSlideover = ({
       leaveTo="w-0 opacity-0"
     >
       <div>
```

Comment:
Div cleanup?

Reply:
I’m not sure about the main purpose of this div and didn’t want to risk breaking the animation. However, I’ll double-check.